apps/nestjs-api/src/reminder-emails/reminder-emails.service.ts

```diff
@@ -59,6 +59,7 @@ export class ReminderEmailsService {
       Profile_First_Approved_At__c: {
         $eq: jsforce.SfDate.toDateLiteral(approvedDate),
       },
+      'Contact__r.ReDI_Active_Mentorship_Matches_Mentee__c': null,
```

Comment:
I think `0` is better than `null` here. AFAIK this is a formula field in Salesforce, so it shouldn't take on other types than `Number`.

Reply:
0 actually didn’t work. That was also my first attempt, I was quite surprised why it was not working and while debugging it I sent myself 650 emails and lost some valuable time on the Salesforce Dev Console.

Reply:
I think this field has a different purpose. I just checked my CON profile on production. I have an ongoing mentorship with Eric, and this field displays 1. 
![image](https://github.com/talent-connect/connect/assets/51786805/46f526f7-d670-4aa8-8f29-d3681b55776f)

So, it correctly shows the `Active_Mentorship_Matches`,  not the number of applications. 

And this is from Eric's mentor CON profile:
![image](https://github.com/talent-connect/connect/assets/51786805/e58be549-32ae-493e-bfc5-d17afe4ae180)

Maybe creating a custom field on the CON profile to check the mentee applications would be better? Plus using the `Active Mentee` field to filter out those with an ongoing mentorship.

Reply:
It's a bit of a mess:
![image](https://github.com/talent-connect/connect/assets/51786805/914be2a2-c60d-4dbc-834c-1e77d0359833)

I didn't find the Roll-Up Summary (COUNT Mentorship Match) for `Active_Mentorship_Matches_Mentee__c`. We definitely need Ghaid's help to determine how we count the same for mentees and create a new count formula field for our purposes.

Reply:
@katamatata But I tested with using `ReDI_Active_Mentorship_Matches_Mentee__c` and when I didn't have any applications vs when I had an application (wiithout accepted), it worked.

Reply:
Hey, so Kate and I dug a bit into this. 

**To summarize**: none of the four fields...
`ReDl_Active_Mentorship_Matches_Mentee`, `ReDI_Total_Mentorship_Matches_Mentee_c`, `Total_Mentorship_Matches_Mentor_c`, `Active_Mentorship_Matches_Mentor_c`
... are referenced in the codebase. It's unclear how three out of four of them are actually updated in Salesforce (since they're just a `Number` field). Our working theory is that a Salesforce "plugin" called Rollup Helper was doing this job, but checking its config, we can't find configuration that ensures such updates. Also, I remember discussing with Manu that the triggers to update the contents of these fields was brittle to begin with.

**Recommendation:** only use `Active_Mentorship_Matches_Mentor_c` in our code. **Don't use** it to count number of applications a mentee has sent. If using that (or other fields) for some reason works, we should either:
1. figure out how it works, then use it if its reliability satisfies us
2. use/configure another field and use that

@helloanil, thoughts?

Reply:
Hey @ericbolikowski, yeah it's a mess that we have all those fields we don't even know how they work 😄. I'm not sure if I can follow your recommendation though: "Only use in our code, not to count number of applications a mentee has sent." You say "use it in code" but then "don't use it".

If we don't feel safe using this field, I'm happy to follow number 2 and configure another field for this.